views/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="new">
+  <form id="newform" action="/create" method="post">
+    <h2>Add new translation:</h2>
+    <table>
+      <tr>
+        <td>File</td>
+        <td>
+          <select name='file_radix'>
+            <% app.store.file_radixes.each do |radix| %>
+              <option value="<%=radix%>"><%= "#{radix}[locale].yml" %></option>
+            <% end %>
+          </select>
+        </td>
+      </tr>
+      <tr>
+        <td>Key</td>
+        <td><input name='key' type="text" size="60" value="<%= req.params['key']%>"/></td>
```

Comment:
We wan't to sanitize this somehow. If I press 'New' and there's a filter, I get this form:

![2015-04-24 at 15 58](https://cloud.githubusercontent.com/assets/252/7327387/b87317f6-ea9a-11e4-96b2-68e4516d2200.png)

We don't want the special characters there.